spec/dummy/package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "dependencies": {
     "@rails/webpacker": "5.1.1",
-    "activeadmin-chat": "1.0.0"
+    "activeadmin-chat": "file:../../javascript"
```

Comment:
Yessss, did you find a way to use this method on production too?

Reply:
I needed to do this change in order to get the dummy app be updated with latest changes.

I didn't try to have this solution for apps including the gem. I guess the solution would be to make the JS files available in the gem but then it would be kinda cumbersome to locate the JS files from the `package.json`, right? e.g. my gems are installed in `~/.gem/ruby/2.7.1/gems/...` so a path like that would be needed to be specified in the `package.json` which doesn't seem convenient